test/browser_tests/cucumber-new/features/suites/apps/ask_cfpb.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Test the Ask CFPB page in English and Spanish
```

Comment:
The rest of the scenarios in this file need some updating, I'm sure, but the ideas are good.

Reply:
I agree there are scenarios that need updating in ask_cfpb feature file.